support-frontend/app/admin/settings/SettingsProvider.scala

```diff
@@ -30,10 +31,17 @@ abstract class SettingsProvider[T] {
 class AllSettingsProvider private (
   switchesProvider: SettingsProvider[Switches],
   amountsProvider: SettingsProvider[AmountsRegions],
-  contributionTypesProvider: SettingsProvider[ContributionTypes]) {
+  contributionTypesProvider: SettingsProvider[ContributionTypes],
+  metricUrl: MetricUrl
+) {
 
   def getAllSettings(): AllSettings = {
-    AllSettings(switchesProvider.settings(), amountsProvider.settings(), contributionTypesProvider.settings())
+    AllSettings(
+      switchesProvider.settings(),
+      amountsProvider.settings(),
+      contributionTypesProvider.settings(),
+      metricUrl
```

Comment:
I did this as a static value as I don't think we will change it without a deploy, as it's an infra thing rather than a user facing setting.  The additional wrapper looked to have a fair bit of ceremony over no special treatment.  We can always wrap it in the appropriate abstraction later.